target/linux/ramips/image/mt7621.mk

```diff
@@ -262,6 +262,15 @@ define Device/totolink_a7000r
 endef
 TARGET_DEVICES += totolink_a7000r
 
+define Device/adslr_g7
+  DTS := ADSLR-G7
+  IMAGE_SIZE := $(ralink_default_fw_size_16M)
+  DEVICE_VENDOR := ADSLR
+  DEVICE_MODEL := G7
+  DEVICE_PACKAGES := kmod-mt7615e wpad-basic default-settings
```

Comment:
I don't remember we have a package called "default-settings". Please drop it.

Reply:
This package has been removedThis package has been removed